src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -290,7 +290,8 @@ class boss_archimonde : public CreatureScript
 
             Enraged = false;
             HasProtected = false;
-            WorldtreeTraget = reinterpret_cast<Creature*>(me->SummonCreature(WORLDTREE_CHANNEL_TARGET, 5503.713f, -3523.436f, 1608.781f, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 360000s));
+            summons.DespawnAll();
+            Creature* WorldtreeTraget = reinterpret_cast<Creature*>(me->SummonCreature(WORLDTREE_CHANNEL_TARGET, 5503.713f, -3523.436f, 1608.781f, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 360000s));
```

Comment:
i wonder if there is any spell doing this.

Reply:
I Think Channel Spell 39140 need a target to cast.Any other idea?

Reply:
if you jump into irc i can share you one sniff.

Reply:
discord.com?

Reply:
you can reach me there too yes.

Reply:
Actually all you have to do here is spawn WORLDTREE_CHANNEL_TARGET in DB and then cast 39140 on self or nullptr(both ways should work but should be tested), ideally store guid of WORLDTREE_CHANNEL_TARGET in instance script and cast spell on it using instance->GetCreature (there's one example in this file)